lib/classes/CLI.test.js

```diff
@@ -157,6 +157,34 @@ describe('CLI', () => {
     });
   });
 
+  describe('#isConfigIndependentRequest()', () => {
+    it('should not load config file if running "config credentials" command', () =>
+      runServerless({
+        cwd: fixtures.map.configTypeYml,
+        cliArgs: [
+          'config',
+          'credentials',
+          '--provider',
+          'aws',
+          '--key',
+          'AKIAZKEY',
+          '--secret',
+          'SECRET',
+        ],
+        // eslint-disable-next-line no-shadow
+      }).then(({ serverless }) => {
```

Comment:
Let's not use ESLint disablers. Test that rely on `runServerless`, should land in last describe block (named `CLI [new tests]`)